tests/serve/test_vllm.py

```diff
@@ -223,6 +260,33 @@ def wait_for_ready(self, payload, logger=logging.getLogger()):
         model="Qwen/Qwen3-0.6B",
         delayed_start=45,
     ),
+    "multimodal_agg": VLLMConfig(
+        name="multimodal_agg",
+        directory="/workspace/examples/multimodal_v1",
+        script_name="agg.sh",
+        marks=[pytest.mark.gpu_2, pytest.mark.vllm],
+        endpoints=["v1/chat/completions"],
+        response_handlers=[
+            chat_completions_response_handler,
+        ],
+        model="llava-hf/llava-1.5-7b-hf",
+        delayed_start=45,
+        extra_args=["--model", "llava-hf/llava-1.5-7b-hf"],
```

Comment:
I wonder if this would be "args" and not extra_args

Reply:
Fixed.